TextOutput.go

```diff
@@ -16,36 +16,42 @@
 
 package dnstap
 
-import "bufio"
-import "io"
 import "log"
-import "os"
 
 import "github.com/golang/protobuf/proto"
+import "github.com/dmccombs/reopen"
```

Comment:
This is a fork of https://github.com/client9/reopen to add `Flush()` support. A PR is open to add that support upstream, but for now uses my fork.